README.md

```diff
@@ -22,6 +22,7 @@ composer remove --dev phpro/grumphp nikic/php-parser phpstan/phpstan friendsofph
 
 ```yaml
 parameters:
+    commit.regex: "/JIRA-([0-9]*)/i"
     phpstan.config: "phpstan.neon"
     phpcsfixer2.config: ".php_cs.dist"
     phpstan.level: 2
```

Comment:
What phpstan level should we aim for (if we decide to stick with phpstan)

Reply:
At least 5. But would aim for 7.

Reply:
Done, it's override-able anyways.

Reply:
Yes : highest quality by default!
Would it be possible to provide presets for both phpstan and psalm - but make one selectable by config params?
I prefer psalm on level 1 (highest)